api/src/kots_app/kots_ffi.ts

```diff
@@ -108,7 +108,7 @@ export async function kotsAppCheckForUpdates(app: KotsApp, currentCursor: string
   }
 }
 
-export async function kotsAppDownloadUpdates(updatesAvailable: Update[], app: KotsApp, stores: Stores): Promise<void> {
+export async function kotsAppDownloadUpdates(updatesAvailable: Update[], app: KotsApp, stores: Stores): Promise<any> {
```

Comment:
Why change this to `any`?  This function doesn't return anything

Reply:
Yeah should still be void. Changing that